src/timer_wrap.cc

```diff
@@ -70,6 +70,7 @@ class TimerWrap : public HandleWrap {
                    AsyncWrap::PROVIDER_TIMERWRAP) {
     int r = uv_timer_init(env->event_loop(), &handle_);
     CHECK_EQ(r, 0);
+    RegisterHandleCleanup(reinterpret_cast<uv_handle_t*>(&handle_));
```

Comment:
Now that all these go though the persistent handle visitor this can probably be dropped and cleanup happen there.